### PR TITLE
Added instructions to add mock group

### PR DESCRIPTION
**Pull request by chkumar246**

### CODE REVIEW

**Comment by chkumar246** [comment]
@hguemar @javierpena Please have a look.

File: source/blog/2016-07-15-how-to-build-new-openstack-packages.html.md [hunk 1]
```diff
@@ -65,7 +71,7 @@ Now your system is ready to use DLRN.
     $ rdopkg info && cp -r ~/.rdopkg/rdoinfo $SOMEWHERE_LOCAL
     $ cd $SOMEWHERE_LOCAL/.rdopkg/rdoinfo
 
-[2.] Edit the rdoinfo.yml file and add package entry in the last
+[2.] Edit the rdo.yml file and add package entry in the last
 
     $ vim rdoinfo.yml
```
Comment (javierpena) [inline source/blog/2016-07-15-how-to-build-new-openstack-packages.html.md]: This should be $ vim rdo.yml

Comment (chkumar246) [inline source/blog/2016-07-15-how-to-build-new-openstack-packages.html.md]: Fixed.

**Comment by javierpena** [comment]
👍

**Pull request merged**